settings.gradle.kts

```diff
@@ -60,3 +53,42 @@ gradleEnterprise {
         publishAlwaysIf(isCiBuild)
     }
 }
+
+@Suppress("UnstableApiUsage") // Central declaration of repositories is an incubating feature
+dependencyResolutionManagement {
+
+    repositoriesMode.set(RepositoriesMode.PREFER_SETTINGS)
+
+    repositories {
+        mavenCentral()
+        google()
+
+        // Declare the Node.js & Yarn download repositories
+        ivy("https://nodejs.org/dist/") {
+            name = "Node Distributions at $url"
+            patternLayout { artifact("v[revision]/[artifact](-v[revision]-[classifier]).[ext]") }
+            metadataSources { artifact() }
+            content { includeModule("org.nodejs", "node") }
+        }
+        ivy("https://github.com/yarnpkg/yarn/releases/download") {
+            name = "Yarn Distributions at $url"
+            patternLayout { artifact("v[revision]/[artifact](-v[revision]).[ext]") }
+            metadataSources { artifact() }
+            content { includeModule("com.yarnpkg", "yarn") }
+        }
+
+        maven("https://www.jetbrains.com/intellij-repository/snapshots")
+        maven("https://www.jetbrains.com/intellij-repository/releases")
+        maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-ide")
+        maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-ide-plugin-dependencies")
+        maven("https://cache-redirector.jetbrains.com/intellij-dependencies")
+        maven("https://www.myget.org/F/rd-snapshots/maven/")
+    }
+
+    pluginManagement {
+        repositories {
+            gradlePluginPortal()
+            mavenCentral()
+        }
+    }
```

Comment:
`pluginManagement` should be placed out of `dependencyResolutionManagement`.

Reply:
haha wow, I've been putting `pluginManagement` in the wrong place for about a year, thanks

Reply:
Ah, if `pluginManagement` is placed outside of `dependencyResolutionManagement` then the block needs to be moved, otherwise there's an error. 

```
dokka/settings.gradle.kts:45:1: Unexpected `plugins` block found. `plugins` can not appear before `pluginManagement`.
> Unexpected `plugins` block found. `plugins` can not appear before `pluginManagement`.
```

Which is okay, I will move it, but personally I think it's neater if all the repositories are together inside one block.

Reply:
It should be at the top of `settings.gradle`.